lib/chat_api_web/router.ex

```diff
@@ -98,7 +98,11 @@ defmodule ChatApiWeb.Router do
     resources("/accounts", AccountController, only: [:update, :delete])
     resources("/messages", MessageController, except: [:new, :edit])
     resources("/conversations", ConversationController, except: [:new, :edit, :create])
-    resources("/customers", CustomerController, except: [:new, :edit, :create])
+
+    resources("/customers", CustomerController, except: [:new, :edit, :create]) do
+      resources("/notes", NoteController, except: [:new, :edit])
```

Comment:
i'd prefer not to nest this resource -- in general, i think it's best to keep as many resources at the top level (i.e. `/api/notes` if possible. (this is how we did it at stripe, and i think this stack overflow comment sums up my feelings nicely: https://stackoverflow.com/a/36410780)

Reply:
V nice! People def have differing opinions about this

Reply:
Works for me!